includes/class-wc-install.php

```diff
@@ -307,8 +307,8 @@ private static function create_cron_jobs() {
 		wp_clear_scheduled_hook( 'woocommerce_tracker_send_event' );
 
 		$ve = get_option( 'gmt_offset' ) > 0 ? '-' : '+';
-
-		wp_schedule_event( strtotime( '00:00 tomorrow ' . $ve . get_option( 'gmt_offset' ) . ' HOURS' ), 'daily', 'woocommerce_scheduled_sales' );
+		
```

Comment:
There is some extra space here now.

Reply:
Huh, must be some formatting from GitHub.com. Let me check it out.